core/logger/manager.php

```diff
@@ -75,9 +82,15 @@ public function rest_error_handler( $error_number, $error_message, $error_file,
 			}
 		}
 
-		$this->shutdown( $error->get_error_data() );
+		$error_data = $error->get_error_data();
+
+		$this->shutdown( $error_data, true );
+
+		return $error_data;
```

Comment:
Do you need this `return`?

Reply:
I think it's fine, you still think it's not, please tell me how to do it :)